### ISSUE TITLE
Add StimAfterPokeOut GUI drop down functionality

### ISSUE THREAD

**Issue by HenryJFlynn** [HenryJFlynn/mouse2afc#15]
- [x] add GUI drop down
     - [x] Install Qt Designer:
          - https://realpython.com/qt-designer-python/#installing-and-running-qt-designer 
          - https://www.pythonguis.com/installation/install-qt-designer-standalone/ 
     - [x] Learn how to use the Qt Designer
          - https://doc.qt.io/qt-6/qtdesigner-manual.html
- [x] add into if statement to choice variable values
- [x] add variables into output_actions

**Comment by HenryJFlynn** [comment]
Useful resources: https://youtube.com/playlist?list=PLzMcBGfZo4-lB8MZfHPLTEHO9zJDDLpYj

**Comment by HenryJFlynn** [comment]
The possible options in a drop down (qt calls them a combobox) are set within a class in the `/definitions` folder. The class is named after what variable is being selected and the choices are defined within: 
```python
from .special_enum import SpecialEnum


class FeedbackDelaySelection(SpecialEnum):
    Fix = 1
    AutoIncr = 2
    TruncExp = 3
    none = 4

```

this class is somehow fed into a function which makes the gui

**Comment by HenryJFlynn** [comment]
idea: add a combobox for StimAfterPokeOut and create a class for it will the options. See if the current code picks it up, hoping that the code was designed robustly.

**Comment by HenryJFlynn** [comment]
Changes I made:

1. Using Qt Designer, I edited the .ui file to change StimAfterPokeOut from a checkbox to a ComboBox. 
2. In the definitions folder, I created a script stim_after_poke_out containing the class StimAfterPokeOut with its given options.
3. In state_matrix.py, I am deleted all logic to do with ChoiceStopStimulus and EWDStopStimulus. They are outdated and will replaced by WaitForDecisionStim, WaitFeedbackStim, and WaitForPokeOutStim.
     - Doing a Ctrl+Shift+F in Matlab and searching for ChoiceStopStimulus and EWDStopStimulus they are only present in outdated versions of the protocol
5. Added logic for WaitForDecisionStim, WaitFeedbackStim, WaitForPokeOutStim
6. added WaitForDecisionStim, WaitFeedbackStim, WaitForPokeOutStim to output_actions of respective states
